swig/python/setup.cfg

```diff
@@ -1,10 +0,0 @@
-# You can override default build options here
```

Comment:
we can't suppress swig/python/setup.cfg, as we also still support non-CMake builds, that will require it to be present. The solution would be to generate swig/python/setup.cfg from setup.cfg.in only in the ${CMAKE_CURRENT_BINARY_DIR} for CMake builds